src/include/catalog/postgres/pg_proc.h

```diff
@@ -4,7 +4,7 @@
 
 #include "catalog/catalog_defs.h"
 
-// TODO(Deepayan): change back to lower value once all builtins are added
+// TODO(dpatra): change back to lower value once all builtins are added
```

Comment:
Could you fix this before we merge it in?

Reply:
I left it set like this for now bc we expect all the PR's to the hackathon to be merged in soon. Once that is done, I'll revert it to a reasonable value.